lib/templates/json_texture.template.js

```diff
@@ -0,0 +1,43 @@
+path = require("path");
+
+function jsonTextureTemplate(data) {
+  var spriteObj = {};
+
+  // Create frame data for each sprite.
+  spriteObj.frames={};
+
+  data.sprites.forEach(function (sprite) {
+    var entry={
+      frame: {
+        "x": sprite.x,
+        "y": sprite.y,
+        "w": sprite.width,
+        "h": sprite.height
+      }
+    };
+
+    spriteObj.frames[path.basename(sprite.source_image)]=entry;
```

Comment:
Can we please use `sprite.name` instead of `path.basename(sprite.source_image)`? This allows for adjustment via `grunt-spritesmith's cssVarMap`. Additionally, please make the spacing consistent with the rest of this repo:

```js
spriteObj.frames[sprite.name] = entry;
```